src/main/java/net/fabricmc/meta/web/ServerBootstrap.java

```diff
@@ -177,4 +182,11 @@ private static void writePropertiesToJar(Path inputJar, Path outputJar, String l
         Files.copy(workingFile, outputJar);
         Files.delete(workingFile);
     }
+
+    private static int getCacheDuration(Context ctx) {
+        if (ctx.pathParamMap().containsValue("stable")) {
+            return 900;
```

Comment:
I think 900 is still too high, people may expect less after a MC release/snapshot? Maybe 120?

Reply:
The two cache duration values that were suggested to me on Discord were 15 minutes and 1 hour. I opted for the shorter one. I agree that "the shorter the better" is the way to go, but don't want to do anything that would hammer the server too much.